app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:likesCounter)
+  end
```

Comment:
- [Optional] COnsider adding `after_save` to call the `update_likes_counter` method every time a new like has been saved to database, then make the update_likes_counter method private. This will save you the stress of manually updating likes counter everytime a like is added.